marquee/TimeDB.cpp

```diff
@@ -82,13 +82,25 @@ time_t TimeDB::getTime()
   JsonObject& root = json_buf.parseObject(jsonArray);
   localMillisAtUpdate = millis();
   Serial.println();
-  if (root["timestamp"] == 0) {
-    return 20;
-  } else {
+  errorMessage = (const char*) root["message"];
```

Comment:
errorMessage should probably be set to "" (empty string) at the top of this method as it will always retain the last error even when the error is resolved.